en/tools/config-catalog-generator/data/configs.json

```diff
@@ -1639,6 +1708,67 @@
                 }
             ],
             "exampleFile": "apim.notification.toml"
-        }
+        },
+        {
+            "title": "Registry indexing configurations",
+            "options": [
+                {
+                    "name": "indexing",
+                    "required": false,
+                    "description": "Index configurations related to meta data search in registry.",
```

Comment:
"metadata" is one word

Reply:
fixed with https://github.com/wso2/docs-apim/pull/2627/commits/3660eecc716fd15449381c42867da33b605f0a1f